apps/showcase/src/components/showcase/forms/forms-pres.component.ts

```diff
@@ -0,0 +1,138 @@
+import { AsyncPipe, CommonModule, formatDate } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input, type OnDestroy, type OnInit, ViewEncapsulation } from '@angular/core';
+import { ReactiveFormsModule, UntypedFormControl } from '@angular/forms';
+import { O3rComponent } from '@o3r/core';
+import { Localization } from '@o3r/localization';
+import { CustomFormValidation } from '@o3r/forms';
+import { Observable, Subject, Subscription } from 'rxjs';
+import { FormsEmergencyContactPresComponent, FormsPersonalInfoPresComponent } from '../../utilities';
+import { EmergencyContact, PersonalInfo } from './contracts';
+import { FormsPresTranslation, translations } from './forms-pres.translation';
+import { dateCustomValidator, formsPresValidatorGlobal } from './forms-pres.validators';
+
+@O3rComponent({ componentType: 'Component' })
+@Component({
+  selector: 'o3r-forms-pres',
+  standalone: true,
+  imports: [
+    AsyncPipe,
+    CommonModule,
+    FormsEmergencyContactPresComponent,
+    FormsPersonalInfoPresComponent,
+    ReactiveFormsModule
+  ],
+  templateUrl: './forms-pres.template.html',
+  styleUrl: './forms-pres.style.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FormsPresComponent implements OnInit, OnDestroy {
+
+  /** Localization of the component */
+  @Input()
+  @Localization('./forms-pres.localization.json')
+  public translations: FormsPresTranslation;
+
+  /** Observable used to notify the component that a submit has been fired from the page */
+  public submitTrigger$: Observable<boolean>;
```

Comment:
This has to be an @Input @AsyncInput received from the page component in order to notify this component to trigger a submit. I am assuming here that we have a button on the page component which triggers the submit, not on this component. 
With the button on this component we don't need the observables to handle the submit, we call directly _submitAction_.
But as we want to showcase the complex example I would move the button at page level and transform the submitTrigger in an Input.

Reply:
The button is in this component, and not on the page component. 

To explain the structure, I have a page which contains the forms component (this current component), which contains two subcomponents (`forms-personal-info` and `forms-emergency-contact`) each having a form. The submit button is in the forms component, but if you prefer I can make the structure more complex and move the button to the page.